File: 167. Two Sum II - Input array is sorted/twosumii.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        left, right = 0, len(numbers)-1
        while not numbers[left] + numbers[right] == target:
            if numbers[left] + numbers[right] < target:
                left += 1
            else:
                right -= 1            
        return [left+1,right+1]

File: 167. Two Sum II - Input array is sorted/twosumii.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int left = 0, right = (int)numbers.size()-1;
        while (numbers[left] + numbers[right] != target){
            if (numbers[left] + numbers[right] < target){
                ++left;
            }
            else{
                --right;
            }
        }
        return vector<int>({left+1,right+1});
    }
};